protobuf/plugin/fieldpath/helpers.go.go

```diff
@@ -14,14 +14,15 @@
    limitations under the License.
 */
 
-package plugin
+package fieldpath
 
 import (
+	"github.com/containerd/containerd/protobuf/plugin"
 	"github.com/gogo/protobuf/proto"
 	"github.com/gogo/protobuf/protoc-gen-gogo/descriptor"
 )
 
 // FieldpathEnabled returns true if E_Fieldpath is enabled
-func FieldpathEnabled(file *descriptor.FileDescriptorProto, message *descriptor.DescriptorProto) bool {
-	return proto.GetBoolExtension(message.Options, E_Fieldpath, proto.GetBoolExtension(file.Options, E_FieldpathAll, false))
+func FieldpathEnabled(file *descriptor.FileDescriptorProto, message *descriptor.DescriptorProto) bool { //nolint
```

Comment:
What was the `//nolint` needed for here? Wondering if we could make the `//nolint` more specific, and add a comment what we're suppressing here;

```go
//nolint: <some linter> // Ignore foo bars
```

Reply:
on it.